README.md

```diff
@@ -49,21 +49,23 @@ The attributes listed below are properly formatted as JSON in [*example.json*](h
 
 >>**\_attempts** (number):  This numeric value represents the start of the range. The range continues to the next highest **\_attempts** of another band.
 
->>**\_correct** (string):  When the questions contained are all correct and complete, this is the id of the next content block.
+>>**\_correct** (string):  When the questions contained are all correct and complete, this is the id of the next content block. Use this property for blocks that only contain presentation components as the other two options can be ignored. Leave blank to use the next block.
 
->>**\_partlyCorrect** (string):  When the questions contained are partly correct and complete, this is the id of the next content block.
+>>**\_partlyCorrect** (string):  When the questions contained are partly correct and complete, this is the id of the next content block. Leave blank to use the next block.
 
->>**\_incorrect** (string):  When the questions contained are all incorrect and complete, this is the id of the next content block.
+>>**\_incorrect** (string):  When the questions contained are all incorrect and complete, this is the id of the next content block. Leave blank to use the next block.
```

Comment:
Leave blank to use the next block.

This is not true. It will not continue if left empty.